tmt/utils/__init__.py

```diff
@@ -348,15 +348,7 @@ def _normalize_fmf(
                 - ppc64
         """
 
-        normalized: FmfContext = FmfContext()
-
-        for dimension, values in spec.items():
-            if isinstance(values, list):
-                normalized[str(dimension)] = [str(v) for v in values]
-            else:
-                normalized[str(dimension)] = [str(values)]
-
-        return normalized
+        return cls.from_serialized(spec)
```

Comment:
Would be good to split these changes to a dedicated PR just to keep the review more concentrated.